yarn.lock

```diff
@@ -2403,65 +2403,65 @@
   dependencies:
     "@babel/runtime" "^7.3.1"
 
-"@luma.gl/constants@8.5.12", "@luma.gl/constants@^8.5.12":
-  version "8.5.12"
-  resolved "https://registry.yarnpkg.com/@luma.gl/constants/-/constants-8.5.12.tgz#4fe916634286f8ad9f41c0e688d75db904b6ea60"
-  integrity sha512-XKW4/Ur6azfoytV7R+84KG5h4E+FwqVOnxAAYyXk8kbNoI6uY829bj+/TxuUV6oJCzflmSCjs1Z4tVbsCVn5Lg==
+"@luma.gl/constants@8.5.13", "@luma.gl/constants@^8.5.13":
```

Comment:
The root `package.json` has `@luma.gl/test-utils": "^8.5.12"`, worth bumping to 8.5.13 also?